packages/uniflash/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=uniflash
+pkgver=8.7.0.4818
+pkgrel=1
+pkgdesc="Universal Flash Programmer for Texas Instruments devices. Provides a single interface for programming Flash memory and executing Flash based operations on supported targets."
+arch=('i686' 'x86_64')
```

Comment:
AL supports only x86_64 and we support only x86_64 and aarch64. I'm not sure that `i686` would be allowed or would work on the build system.